### ISSUE TITLE
[Question]: How to clean all empty stubs after `flow-typed install` ?

### ISSUE THREAD

**Issue by bacloud23** [flow-typed/flow-typed#4572]
### What do you want to know?

Hi, 
I've started using flow and little by little I'm really liking the power of it.

Now I generated stubs for all dependencies, but knowing that there are mostly empty definitions, I understand that the purpose is to type them all. but this is unpractical for most projects. I think it is more normal to expect users of this tool to only use already defined **libdefs** rather than to expect everyone using `flow-typed` to provide more definitions.

The major problem because of it, I'm asking, is that I'm using *webstorm* which supports `// @flow` fine, but after generating stubs, I lost all auto completion, so instead of indexing definitions from `node_module` it is looking in `flow-typed` folder, which is expected. 

To sum up, it would be better to have only provided **libdefs** when you run `flow-typed install`, or to 

Kindly <3

**Comment by bacloud23** [comment]
ok my bad, I missed `flow-install -s`

**Comment by Brianzchen** [comment]
Hi, I'm glad you're enjoying flow, it's an extremely powerful tool to add to your javascript projects. To start I want to make a very strong suggestion that you give VSCode instead of webstorm a go because iirc features like https://flow.org/en/docs/types/mapped-types/ can't be parsed correctly and you'll get lots of errors on your text editor, at least not with IntelliJ which a colleague of mine was using prior.

Anyways back to your question, can you please provide
1. What version of flow-typed are you on, `flow-typed --version`
2. Can you please send us your output when you run `flow-typed install`? It should print out every dependency you are using and whether it was able to find real definition and install it.

In thinking about this, is `ioredis` specified in your `package.json` dependencies? Or are you depending on it as a transitive dependency in which case `flow-typed` does not download these libdefs for you intentionally.

**Comment by bacloud23** [comment]
Thank you for your response
